UnityGLTF/Assets/Resources/GLTFSceneTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using System.Collections;
+using UnityGLTF;
+using UnityGLTF.Examples;
+using System.IO;
+using System.Linq;
+using UnityEngine.Rendering;
+using System.Collections.Generic;
+using System;
+using Object = System.Object;
+using Newtonsoft.Json;
+using AssetGenerator;
+using NUnit.Compatibility;
+using UnityEngine.EventSystems;
+using static AssetGenerator.Manifest;
+using Camera = UnityEngine.Camera;
+using UnityEditor.SceneManagement;
+using UnityEngine.SceneManagement;
+
+public class GLTFSceneTests
+{
+    static string GLTF_SCENES_PATH = Application.dataPath + "./UnityGLTF/Examples/";
+    static string GLTF_SCENARIO_OUTPUT_PATH = Application.dataPath + "/../ScenarioScenesTests/Output/";
+    private static bool GENERATE_REFERENCEDATA = false;
+ 
+    private Dictionary<string, AssetGenerator.Manifest.Camera> cameras =
+        new Dictionary<string, AssetGenerator.Manifest.Camera>();
+
+ 
+    public static IEnumerable<string> SceneFilePaths
+    {
+        get
+        {
+            Directory.CreateDirectory(GLTF_SCENARIO_OUTPUT_PATH);
+            var gltfPathUri = new Uri(GLTF_SCENES_PATH);
+            string[] files = Directory.GetFiles(GLTF_SCENES_PATH, "*.unity", SearchOption.AllDirectories);
+            for (int i = 0; i < files.Length; i++)
+            {
+                string filename = Path.GetFileNameWithoutExtension(files[i]);
+                var fileUri = new Uri(files[i]);
+                files[i] = gltfPathUri.MakeRelativeUri(fileUri).ToString();
+
+            }
+            return files;
+        }
+    }
+
+    [UnityTest]
+    public IEnumerator GLTFScenarios([ValueSource("SceneFilePaths")] string scenePath)
+    {
+	    SceneManager.LoadScene(Path.GetFileNameWithoutExtension( scenePath));
+       
+	    //wait one frame for loading to complete
+        yield return null;
+        ////GLTFComponent gltfcomponent = ActiveGLTFObject.AddComponent<GLTFComponent>();
```

Comment:
Why commented out?

Reply:
the gltf object is part of the scene instead of created on the fly to simplify location relative to camera. removing